### PR TITLE
Changed href in DocumentCard basic example to bing

### PR DESCRIPTION
**Pull request by mirajp**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes
Changed href in example to bing.com

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by Vitalius1** [comment]
@mirajp looks like you made these changes directly on your local master:
screen shot 2019-01-14 at 12 29 11 pm
 Not sure how this will affect your local clone after this is merged.

Thanks for the fix.

**Comment by mikewheaton** [comment]
@mirajp Could you sign the license agreement [above](https://github.com/OfficeDev/office-ui-fabric-react/pull/7638#issuecomment-454114016)? It's a one time step that will let you contribute from here forward.

**Comment by Vitalius1** [comment]
@mirajp for my comment above you can find our [WIKI](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Bug-Fixes) very helpful.

File: common/changes/office-ui-fabric-react/documentcard-basic-example-href-change_2019-01-14-18-37.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Changed href in basic example to bing.com",
+      "type": "patch"
```
Comment (Vitalius1) [inline common/changes/office-ui-fabric-react/documentcard-basic-example-href-change_2019-01-14-18-37.json]: Also, the type here should be `none` as this is an example change that should not trigger a release. @micahgodbolt can you confirm on my suggestion?
```suggestion
      "type": "none"
```

File: common/changes/office-ui-fabric-react/documentcard-basic-example-href-change_2019-01-14-18-37.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Changed href in basic example to bing.com",
```
Comment (Vitalius1) [inline common/changes/office-ui-fabric-react/documentcard-basic-example-href-change_2019-01-14-18-37.json]: Instead maybe suggest a change for the comment itself 😄 
```suggestion
      "comment": "DocumentCard: changed href in basic example to bing.com",
```

**Pull request merged**